src/pages/ImporterPage/ImporterPage.js

```diff
@@ -22,6 +22,20 @@ function getLink(appname, credentials) {
 	}
 	return `${IMPORTER_LINK}${JSON.stringify(parameters)}&header=false`;
 }
+
+function getUrlParams(url) {
+	if (!url) {
+		return {};
+	}
+	const searchParams = new URLSearchParams(url);
```

Comment:
We should use `url-search-params` package here. [Refer this](https://github.com/appbaseio/reactivesearch/blob/dev/packages/web/src/components/basic/ReactiveBase.js#L16).

Reply:
Done